### PR TITLE
doc: improve the "Pull Request" feature documentation.

### PR DESCRIPTION
**Pull request by javierron**
I'm not sure how much detail we should put here (e.g. name of storage account, name of table, web service details)

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -83,7 +83,9 @@ The documentation about the scripts is [available here](scripts.md). More detail
 
 ### Pull-requests
 
-When Repairnator is configured to create a pull-request automatically, it uses the text of `TEXT_PR` from [AbstractRepairStep](https://github.com/eclipse/repairnator/blob/master/src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/AbstractRepairStep.java). In the future, it is planned to generate a specific piece of text to explain the patch (see [Explainable Software Bot Contributions: Case Study of Automated Bug Fixes](http://arxiv.org/pdf/1905.02597) ([doi:10.1109/BotSE.2019.00010](https://doi.org/10.1109/BotSE.2019.00010)))
```
Comment (monperrus) [inline doc/README.md:86]: add a link to where this configuration is explained

File: doc/README.md [hunk 2]
```diff
@@ -83,7 +83,9 @@ The documentation about the scripts is [available here](scripts.md). More detail
...
+When Repairnator is configured to create a pull-request automatically, it uses the text found on [this file](https://github.com/eclipse/repairnator/blob/master/src/repairnator-pipeline/src/main/resources/R-Hero-PR-text.MD). In the future, it is planned to generate a specific piece of text to explain the patch (see [Explainable Software Bot Contributions: Case Study of Automated Bug Fixes](http://arxiv.org/pdf/1905.02597) ([doi:10.1109/BotSE.2019.00010](https://doi.org/10.1109/BotSE.2019.00010)))
+
+By default, the links created in the PR text direct to a web service, which stores the feedback information on a table on our Azure storage account.
```
Comment (monperrus) [inline doc/README.md:88]: per our discussion, we create a new repo in the repairnator org and we link to it here

File: doc/README.md (hunk 1 above)
Comment (javierron) [reply doc/README.md:86]: done!

Comment (javierron) [reply doc/README.md:86]: new repo: https://github.com/repairnator/r-hero-feedback-service

**Comment by monperrus** [comment]
Thanks @javierron

**Pull request merged**